### PR TITLE
http timeouts, v1.2.2

### PR DESCRIPTION
**Pull request by nilsbehlen**

### CODE REVIEW

File: src/main/java/org/privacyidea/Endpoint.java [hunk 1]
```diff
@@ -133,15 +134,8 @@ void sendRequestAsync(String endpoint, Map<String, String> params, Map<String, S
...
-                               {
-                                   String encValue = URLEncoder.encode(value, StandardCharsets.UTF_8.toString());
-                                   urlBuilder.addQueryParameter(key, encValue);
-                               }
-                               catch (UnsupportedEncodingException e)
-                               {
-                                   e.printStackTrace();
-                               }
+                               String encValue = URLEncoder.encode(value, StandardCharsets.UTF_8);
+                               urlBuilder.addQueryParameter(key, encValue);
```
Comment (lukasmatusiewicz) [inline src/main/java/org/privacyidea/Endpoint.java:138]: Isn't toString() needed here anymore?

File: src/main/java/org/privacyidea/Endpoint.java [hunk 2]
```diff
@@ -168,14 +162,7 @@ void sendRequestAsync(String endpoint, Map<String, String> params, Map<String, S
...
                                    {
-                                       try
-                                       {
-                                           encValue = URLEncoder.encode(value, StandardCharsets.UTF_8.toString());
-                                       }
-                                       catch (UnsupportedEncodingException e)
-                                       {
-                                           privacyIDEA.error(e);
-                                       }
+                                       encValue = URLEncoder.encode(value, StandardCharsets.UTF_8);
```
Comment (lukasmatusiewicz) [inline src/main/java/org/privacyidea/Endpoint.java:165]: toString()?

**Review by lukasmatusiewicz — CHANGES_REQUESTED** [review]

File: src/main/java/org/privacyidea/Endpoint.java (hunk 1 above)
Comment (nilsbehlen) [reply src/main/java/org/privacyidea/Endpoint.java:136]: `URLEncoder.encode()` can take a `Charset` as second parameter as well

File: src/main/java/org/privacyidea/Endpoint.java (hunk 2 above)
Comment (nilsbehlen) [reply src/main/java/org/privacyidea/Endpoint.java:163]: `URLEncoder.encode()` can take a `Charset` as second parameter as well

Comment (nilsbehlen) [reply src/main/java/org/privacyidea/Endpoint.java:163]: https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/URLEncoder.html#encode(java.lang.String,java.nio.charset.Charset)

**Review by lukasmatusiewicz — APPROVED** [review]

**Pull request merged**